src/app_service/service/stickers/service.nim

```diff
@@ -126,7 +126,7 @@ QtObject:
       buyToken = BuyToken(packId: packId, address: address, price: price)
       buyTxAbiEncoded = stickerMktContract.methods["buyToken"].encodeAbi(buyToken)
     approveAndCall = ApproveAndCall[100](to: stickerMktContract.address, value: price, data: DynamicBytes[100].fromHex(buyTxAbiEncoded))
-    self.eth_service.buildTokenTransaction(address, sntContract.address, gas, gasPrice, isEIP1559Enabled, maxPriorityFeePerGas, maxFeePerGas)
+    ens_utils.buildTokenTransaction(address, sntContract.address, gas, gasPrice, isEIP1559Enabled, maxPriorityFeePerGas, maxFeePerGas)
```

Comment:
Maybe ens_utils isn't right. We might need an `eth_utils` or tx_utils`

Reply:
@jrainville yes, I agree, we may find better place for that, but as I the same comment is for this as I added [here](https://github.com/status-im/status-desktop/pull/4490#discussion_r791061179). `buildTokenTransaction` may/will be used by multiple places, so wi will find better place for it, actually  `ens`, `wallet`, `transaction`, `eth` services will need a final touch at the end.